Makefile

```diff
@@ -32,10 +32,14 @@ test-integration: ## Run the integration testsuite.
 	composer test:integration --ansi
 
 test-matrix:
+	docker run --rm -t -v $$(pwd):/opt/graze/queue -w /opt/graze/queue php:5.5-cli \
```

Comment:
Leave this bit to me, queue needs some general updates anyway. Just need to fix the failing build